automol/vmat.py

```diff
@@ -66,36 +76,33 @@ def from_data(symbs: tuple[str, ...], key_mat, name_mat=None, one_indexed=None):
 
 # # V-Matrix/V-Matrix common functions (document these as z-matrix functions)
 # # # getters
-def symbols(vma, idxs: list[int] | None = None) -> list[str]:
+def symbols(vma: VMatrix, idxs: list[int] | None = None) -> list[str]:
```

Comment:
The return type should be `->  tuple[Symbol, ...]` (equivalent to `tuple[str, ...]`).